subprojects/parseq/src/main/java/com/linkedin/parseq/trace/ShallowTraceBuilder.java

```diff
@@ -124,6 +124,9 @@ public ShallowTraceBuilder setResultType(final ResultType resultType) {
     return this;
   }
 
+  /**
+   * Deprecated. Use {@link #setNativeStartNanos(long)} instead
```

Comment:
we need to use @deprecated to generate proper javadoc for this.https://docs.oracle.com/javase/8/docs/technotes/guides/javadoc/deprecation/deprecation.html